README.md

```diff
@@ -1 +1,40 @@
-# Hello-world
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+# Hello-world
+
+> Description the project.
+
+
+## Built With
+
+- HTML
+- CSS
+
+## Live Demo (if available)
+
+[Live Demo Link](https://livedemo.com)
+
+
+## Getting Started
+
+**Go to the a repository on [github](https://github.com/anasbassi/Hello-world.git).**
+**Fork the repository to your own github."" 
+**Copy the https link and clone it on your local device.**
+
+To get a local copy up and running follow these simple example steps.
```

Comment:
- [ ] Good job :100:  so far by adding this important section to your documentation. So, I noticed that your details are not still enough. I suggest you add some details on how to get the project locally, as for cloning the project. You can add some hints/processes on how to clone the project.

Reply:
I added some steps on how to clone the project on the local device